packages/kotti-ui/source/kotti-avatar/KtAvatar.vue

```diff
@@ -29,13 +29,14 @@ const useTooltip = (name: Ref<string | null>) => {
 
 	useTippy(
 		tooltipTriggerRef,
-		computed(() => ({
+		computed<Partial<TippyProps>>(() => ({
```

Comment:
locally check this with @FlorianWendelborn right now

`useTippy` accepts the 2nd argument (`options`) to be `Ref<Partial<Props>>`


can you elaborate on why this was necessary?